### ISSUE TITLE
Clarify MozFest vs. Mozilla Festival

### ISSUE THREAD

**Issue by edrushka** [MozillaFoundation/2016.Mozillafestival.org#27]
Are we moving towards a more consistent use of MozFest this year (and slowly sunsetting Mozilla Festival)? It would be good to have clarity for webpage and comms copy.

cc @Saallen @KevZawacki

**Comment by KevZawacki** [comment]
Agreed that we should be consistent and use just one! As for which one...

**Comment by edrushka** [comment]
My understanding from the exec level was that we should move towards MozFest, hence acquiring the mozfest.org URL earlier this year.

**Comment by KevZawacki** [comment]
👍 All instances of "Mozilla Festival" in the comms plan have been smote!

**Comment by Saallen** [comment]
Bey, before we press go on this we must run this by the three stakeholders. We talked about this last year but Mark said he would be down to Gunner, Michelle and himself. Mark is pro MozFest change over. I have emailed them and am waiting on a reply

**Comment by Saallen** [comment]
I did not intent to call you pair my "beys" ala beyonce- should of had said Hey ...beys

**Comment by edrushka** [comment]
@Saallen this is something we can address if we get the stakeholders meeting in London. Also, noting that if we decide to use MozFest exclusively, we'll need to tweak the logo/wordmark. This is not in scope for June 1, so we have some time to decide, and transition over the coming months, if that's the decision.

**Comment by Saallen** [comment]
Hi- yes, this move from Mozilla festival to mozfest has been brought up in conversation before but there are 3 key stakeholders who have to agree before any major changes can be made. I am happy for all comms to reflect the name "mozfest " but v reluctant as of yet to discuss logo changes or removing Mozilla from the title just yet. Yes we can chat about this at Stakeholders meeting but logo/watermark might be a 2017 change instread of this year

**Comment by edrushka** [comment]
Sounds good. We can favour 'MozFest' for the website and comms materials this year, saving the final decision for the stakeholders meeting with implementation -- if a decision is made -- to happen in 2017.
